components/Request/Request.jsx

```diff
@@ -8,6 +8,7 @@ import { REDUCER_KEY } from '../../reducers';
 
 const Request = ({ error, matchId, loading, progress, dispatchRequest, dispatchMatchId }) => {
   function submit() {
+    document.getElementById('match_id').value = '';
```

Comment:
Yeah, you're right. I was not sure about this part, but I wanted to clear the value because the overlap from the animation is jarring to look at. If I simply dispatchMatchId it won't re-render and change the 'value' attribute in the TextField though, as far as I'm aware. What do you have in mind?